llvm/test/CodeGen/RISCV/rvv/fixed-vectors-fp-interleave.ll

```diff
@@ -238,26 +238,44 @@ define <64 x half> @interleave_v32f16(<32 x half> %x, <32 x half> %y) {
 define <64 x float> @interleave_v32f32(<32 x float> %x, <32 x float> %y) {
 ; V128-LABEL: interleave_v32f32:
 ; V128:       # %bb.0:
+; V128-NEXT:    addi sp, sp, -16
```

Comment:
This was the diff I was originally concerned by, but thinking about it further, I think this is fine.  All of the increases in register pressure happen in cases where we're legalizing a shuffle which is more than m8.  Such shuffles aren't going to be real common in auto-vectorized code.